src/maker/server.rs

```diff
@@ -133,6 +132,10 @@ fn network_bootstrap(maker: Arc<Maker>) -> Result<Option<Child>, MakerError> {
         maker_address
     );
 
+    maker
+        .as_ref()
+        .track_and_update_unconfirmed_fidelity_bonds()?;
```

Comment:
As this can take some time, the "Server is listening" log should come after this.